### PR TITLE
Product Brand module added same as odoo8(migrated from OCA/ecommarce Branch)

### PR DESCRIPTION
**Pull request by Amit-Parik**

### CODE REVIEW

File: product_brand/product_brand.py [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+
+    def get_pricelist(self):
+        return get_pricelist()
+
+    @http.route([
+        '/shop', '/shop/page/<int:page>',
+        '/shop/category/<model("product.public.category"):category>',
+        '/shop/category/<model("product.public.category"):category>/page/<int:page>'
+        ], type='http', auth="public", website=True)
+    def shop(self, page=0, category=None, search='', **post):
```
Comment (oscarolar) [inline product_brand/product_brand.py:146]: The same  thing as V8 module, here you are breaking inheritance in all methods, please use super.

**Comment by oscarolar** [comment]
Please ensure that all methods used in the controllers are not breaking inheritance, this also happened in this same module for V8.

Best regards.

**Comment by rafaelbn** [comment]
This module must be in https://github.com/OCA/product-attribute/ like https://github.com/OCA/product-attribute/tree/8.0/product_brand
The one must be here is https://github.com/OCA/e-commerce/tree/9.0/website_sale_product_brand

**Comment by rafaelbn** [comment]
Hi @Amit-Parik , thanks for contribution. Please review comments in order to set this PR as for reviewing. Thanks!

**Comment by Amit-Parik** [comment]
ok sure.

**Pull request closed without merging**